src/lib.rs

```diff
@@ -16,19 +16,21 @@
 //!
 //! #[cfg(feature = "sled-storage")]
 //! fn main() {
-//!     let storage = SledStorage::new("data/doc-db").unwrap();
+//!     let storage = SledStorage::new("data.db").unwrap();
```

Comment:
It would be better to be `data/doc-db`.
Currently, `/data` is in `.gitignore` but not `data.db`.
It can be bit inconvenient because `data.db` will appear per each `cargo test` and we need to ignore it.